src/main/java/roomescape/util/CustomDateTimeFormat.java

```diff
@@ -0,0 +1,8 @@
+package roomescape.util;
+
+import java.time.format.DateTimeFormatter;
+
+public class CustomDateTimeFormat {
+    public static final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
```suggestion
    public static final DateTimeFormatter dateFormatter = DateTimeFormatter.ISO_LOCAL_DATE;
```

이 부분이 iso 라고 하는 시간 표현 형식과 동일한 것 같아서 이 부분을 위처럼 변경해보면 어떨까요?
시간 포맷에 대해서도 학습해보시면 좋을 것 같아요
The ISO date formatter that formats or parses a date without an offset, such as '2011-12-03'.
This returns an immutable formatter capable of formatting and parsing the ISO-8601 extended local date format. The format consists of: